finalPrep/README.md

```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+ ![Screen Shot 2022-04-20 at 3 27 29 PM](https://user-images.githubusercontent.com/102986835/164326513-944c9e71-9cb7-4f2a-a8df-5a1c3b2e6143.png)
+ ![Screen Shot 2022-04-20 at 3 28 09 PM](https://user-images.githubusercontent.com/102986835/164326863-469a96ee-4bcf-461f-b603-6b945ac063e0.png)
+ ![Screen Shot 2022-04-20 at 3 28 24 PM](https://user-images.githubusercontent.com/102986835/164326919-e296f004-c325-4e81-9760-70e388b2d812.png)
+
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Mentorship Gist](https://gist.github.com/NickLiggett/3fdf43993db7fdba050be81558367645)
```

Comment:
This will set you up for a great start to your relationship with your mentor. I hope you can keep up your Wednesday Night D&D event, too ☺️